concepts/case/about.md

```diff
@@ -0,0 +1,129 @@
+# Case
+
+[Case][case] (often referred to as switch in other languages) is a form of control expression like if-else.
+Case allows for chaining of multiple if-else-if statements and can be more readable and also allows for powerful constructs.
+
+A case is defined by the keyword `case` followed by an optional expression.
+Then for each case, the keyword `when` is used followed by an expression which is compared to the case expression.
+The `when` keyword should not be indented from the `case` keyword.
+After the `when` keyword is the code that should be executed if the case expression matches the when expression.
+Case allows for an optional `else` statement which is executed if no other case matches.
+
+The case expression is evaluated and then compared to each `when` expression.
+The expression is compared using the case subsumption operator (`===`).
+
+```ruby
+case 1
+when 1
+  "One"
+when 2
+  "Two"
+else
+  "Other"
+end
+# => "One"
+
+# This is the same as:
+tmp = 1
+if 1 === tmp
+  "One"
+elsif 1 === tmp
+  "Two"
+else
+  "Other"
+end
```

Comment:
```suggestion
# This is the same as:
value = 1
if 1 === value
  "One"
elsif 1 === value
  "Two"
else
  "Other"
end
```

Reply:
I would argue that the two code examples are not the same though, where 2 is explicitly matched, while the if statement will never get to the `else`.

Reply:
It is a type on line 30 if it is that you are referring to?

Reply:
Oh, a "typographical" error, yeah, I believe so.  I know that it is a logic error as well, though, did not identify that it might be a typo.